src/base/static/js/routes.js

```diff
@@ -104,7 +104,7 @@ Shareabouts.Util = Util;
 
       // Load the default page when there is no page already in the url
       if (Backbone.history.getFragment() === "") {
-        startPageConfig = S.Util.findPageConfig(options.pagesConfig, {
+        const startPageConfig = _.findWhere(this.options.navBarConfig, {
```

Comment:
I think this is a bug (`this.options` -> `options`):

```suggestion
        const startPageConfig = _.findWhere(options.navBarConfig, {
```